base/sparse/sparsevector.jl

```diff
@@ -1681,3 +1681,35 @@ droptol!(x::SparseVector, tol, trim::Bool = true) = fkeep!(x, (i, x, tol) -> abs
 
 dropzeros!(x::SparseVector, trim::Bool = true) = fkeep!(x, (i, x, other) -> x != 0, nothing, trim)
 dropzeros(x::SparseVector, trim::Bool = true) = dropzeros!(copy(x), trim)
+
+function _fillnonzero!{Tv,Ti}(arr::SparseMatrixCSC{Tv, Ti}, val)
+    m, n = size(arr)
+    arr.colptr = convert(Vector{Ti}, collect(1:m:n*m+1))
+    arr.rowval = convert(Vector{Ti}, vcat([1:m for _ in 1:n]...))
```

Comment:
You want things to be inplace for another epsilon of performance or is there another reason?

Where should this fill! be? It needs to be at a point, where SparseVector, SparseMatrixCSC type is already defined?